xbmc/video/windows/GUIWindowVideoBase.cpp

```diff
@@ -738,7 +738,7 @@ bool CGUIWindowVideoBase::OnItemInfo(int iItem)
 
   OnItemInfo(item.get(), scraper);
 
-  return true;
+  return item->HasVideoInfoTag();
```

Comment:
Wouldn't it make more sense for the above call to `OnItemInfo()` to return a boolean value whether the info dialog has been opened or not and then we could simply forward that return value? This change is not very intuitive IMO.